src/common/ceph_crypto.cc

```diff
@@ -28,7 +27,7 @@ void ceph::crypto::init(CephContext *cct)
 {
 }
 
-void ceph::crypto::shutdown()
+void ceph::crypto::shutdown(bool shared)
```

Comment:
It's needed since other functions will call it with parameter. 
https://github.com/ceph/ceph/blob/master/src/common/ceph_context.cc#L663